ruby/vampires.rb

```diff
@@ -1,8 +1,48 @@
+puts "How many employees will be processed?"
+employees = gets.chomp.to_i
+
+counter = 0
+while counter < employees do
 puts "What is your name?"
-name = gets.chomp
+user = gets.chomp.downcase
 puts "How old are you?"
-age = gets.chomp
+age = gets.chomp.to_i
+puts "What year were you born?"
+year = gets.chomp.to_i
 puts "Our company cafeteria serves garlic bread. Should we order some for you? (Y/N)"
-bread = gets.chomp
+bread = gets.chomp.downcase
 puts "Would you like to enroll in the company's health insurance? (Y/N)"
-insurance = gets.chomp
+insurance = gets.chomp.downcase
+
+actual_age = 2017 - year

```

Comment:
When checking something like a year consider an approach that keeps us from needing to change our code every new year.

Take a look the Ruby `Time` class.  `Time.new.year`